ambry-named-mysql/src/main/java/com/github/ambry/named/MySqlNamedBlobDb.java

```diff
@@ -156,157 +246,92 @@ public CompletableFuture<NamedBlobRecord> get(String accountName, String contain
     TransactionStateTracker transactionStateTracker =
         new GetTransactionStateTracker(remoteDatacenters, localDatacenter);
     return executeTransactionAsync(accountName, containerName, true, (accountId, containerId, connection) -> {
-      try (PreparedStatement statement = connection.prepareStatement(GET_QUERY)) {
-        statement.setInt(1, accountId);
-        statement.setInt(2, containerId);
-        statement.setString(3, blobName);
-        try (ResultSet resultSet = statement.executeQuery()) {
-          if (!resultSet.next()) {
-            throw buildException("GET: Blob not found", RestServiceErrorCode.NotFound, accountName, containerName,
-                blobName);
-          }
-          String blobId = Base64.encodeBase64URLSafeString(resultSet.getBytes(1));
-          Timestamp expirationTime = resultSet.getTimestamp(2);
-          Timestamp deletionTime = resultSet.getTimestamp(3);
-          long currentTime = System.currentTimeMillis();
-          if (compareTimestamp(expirationTime, currentTime) <= 0 && !includeExpiredOptions.contains(option)) {
-            throw buildException("GET: Blob expired", RestServiceErrorCode.Deleted, accountName, containerName,
-                blobName);
-          } else if (compareTimestamp(deletionTime, currentTime) <= 0 && !includeDeletedOptions.contains(option)) {
-            throw buildException("GET: Blob deleted", RestServiceErrorCode.Deleted, accountName, containerName,
-                blobName);
-          } else {
-            return new NamedBlobRecord(accountName, containerName, blobName, blobId, timestampToMs(expirationTime));
-          }
+      long startTime = System.currentTimeMillis();
+      NamedBlobRecord record, recordV2;
+      record = run_get(accountName, containerName, blobName, option, accountId, containerId, connection);
+      try {
+        recordV2 = run_get_v2(accountName, containerName, blobName, option, accountId, containerId, connection);
+        if (!record.equals(recordV2)) {
+          logger.warn("NAMED GET: Data Inconsistence: old record='{}', new record='{}'", record, recordV2);
```

Comment:
for list, I suggest we don't add a warn log. List is not like a point lookup query. For get, warn log message makes sense. If we see two records are different in old and new table, this is a warning level error. But in list, before we backfill, we are going to see lots of differences.

Reply:
I just removed the warn log in LIST part.